rfc/rfc006-distributed-registry.md

```diff
@@ -333,6 +333,13 @@ The care organisation refers to it:
 }
 ```
 
+### 4.1. Network Endpoint Service
+
+The underlying peer-to-peer network does not automatically discover new peers but it relies on the system to tell
+which peers to connect to instead. Nodes SHOULD publish their Nuts Network endpoint(s) as a service in their DID Document(s).
+The `type` of this service MUST be `nuts-network-grpc`. The `serviceEndpoint` MUST be a single gRPC endpoint other
+nodes can use to connect to the node. In MUST be in the format of `<host>:<port>`. A node MAY publish multiple network endpoints.
```

Comment:
Since the service datastructer is very generic, I think it helpfull to add an example service with one or multiple endpoints, or describe the format better as an array of maps?

Reply:
As you say, the datastructure is very generic, that's why I spec'd it as very simple and strict: MUST be a single gRPC endpoint. But I can add an example though.

Reply:
I see I don't honor the compound service spec as stated above in `4.` so I'll add that.